src/ngraph/runtime/ie/ie_executable.cpp

```diff
@@ -0,0 +1,165 @@
+#include "ngraph/pass/manager.hpp"
+#include "ngraph/pass/opset1_upgrade.hpp"
+#include "ngraph/runtime/ie/ie_tensor.hpp"
+#include "ngraph/shape.hpp"
+#include "ngraph/type/element_type.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+namespace
+{
+    InferenceEngine::Blob::Ptr fill_blob(InferenceEngine::SizeVector shape,
+                                         const void* data,
+                                         size_t data_size,
+                                         const element::Type& elem_type)
+    {
+        InferenceEngine::Layout layout;
+        switch (shape.size())
+        {
+        case 1: layout = InferenceEngine::Layout::C; break;
+        case 2: layout = InferenceEngine::Layout::NC; break;
+        case 3: layout = InferenceEngine::Layout::CHW; break;
+        case 4: layout = InferenceEngine::Layout::NCHW; break;
+        case 5: layout = InferenceEngine::Layout::NCDHW; break;
+        case 6: layout = InferenceEngine::Layout::GOIDHW; break;
+        default: THROW_IE_EXCEPTION << "Can't convert dims " << shape.size() << " to Layout!";
+        }
+
+        InferenceEngine::MemoryBlob::Ptr blob;
+
+#define MAKE_IE_TBLOB(type_, precision_, shape_, layout_)                                          \
+    make_shared<InferenceEngine::TBlob<type_>>(                                                    \
+        InferenceEngine::TensorDesc{InferenceEngine::Precision::precision_, shape_, layout_})
+
+        switch (elem_type)
+        {
+        case element::Type_t::f32:
+            blob = MAKE_IE_TBLOB(float, FP32, shape, layout);
+            break;
+        // case element::Type_t::f16: blob = MAKE_IE_TBLOB(float, FP16, shape, layout); break;
+        case element::Type_t::i16: blob = MAKE_IE_TBLOB(int16_t, I16, shape, layout); break;
+        case element::Type_t::u8: blob = MAKE_IE_TBLOB(uint8_t, U8, shape, layout); break;
+        case element::Type_t::i8: blob = MAKE_IE_TBLOB(int8_t, I8, shape, layout); break;
+        case element::Type_t::u16: blob = MAKE_IE_TBLOB(uint16_t, U16, shape, layout); break;
+        case element::Type_t::i32: blob = MAKE_IE_TBLOB(int32_t, I32, shape, layout); break;
+        case element::Type_t::i64: blob = MAKE_IE_TBLOB(int64_t, I64, shape, layout); break;
+        case element::Type_t::u64: blob = MAKE_IE_TBLOB(uint64_t, U64, shape, layout); break;
+        case element::Type_t::boolean: blob = MAKE_IE_TBLOB(uint8_t, BOOL, shape, layout); break;
+        default: THROW_IE_EXCEPTION << "Can't convert type " << elem_type << " to IE Precision!";
+        }
+
+#undef MAKE_IE_TBLOB
+
+        blob->allocate();
+        uint8_t* blob_ptr = blob->rwmap().as<uint8_t*>();
+        memcpy(blob_ptr, data, data_size * elem_type.size());
+        return blob;
+    }
+}
+
+runtime::ie::IE_Executable::IE_Executable(shared_ptr<Function> func, string device)
+{
+    const auto& opset = get_opset1();
+    pass::Manager passes;
+    passes.register_pass<pass::Opset1Upgrade>();
```

Comment:
Can we use `GetOutputElementElimination` pass here and skip checking if node is `op::GetOutputElement::type`?

Reply:
Currently IE is running `GetOutputElementElimination` pass anyway, so I thought it's not necessary.